cmp/grid/GridModel.js

```diff
@@ -87,22 +94,6 @@ export class GridModel {
     /** @member {string[]} */
     @observable groupBy = null;
 
-    /** @member {boolean} */
-    @bindable compact;
-    /** @member {boolean} */
-    @bindable rowBorders;
-    /** @member {boolean} */
-    @bindable stripeRows;
-    /** @member {boolean} */
-    @bindable showHover;
-    /** @member {boolean} */
-    @bindable showCellFocus;
-
-    /** @member {GridApi} */
-    @observable.ref agApi = null;
-    /** @member {ColumnApi} */
-    @observable.ref agColumnApi = null;
```

Comment:
Should we add pass-through getters for agApi and agColumnApi? They were part of the public API, and saving references to these two is a useful feature of GridModel. Any reason not to do so, i.e. do we think it's important that the developer realize these are pulled from the agGridModel?

(We could use also those in the implementation code below FWIW.)